### DISCUSSION TITLE
windows 10 support

### DISCUSSION THREAD

**Discussion by Virgilio-AI** [andmarti1424/sc-im#511] [Q&A]
I have seen this repo and looks fantastic, I am currently using nvim in windows 10 and I would love to have sc-im, is there a way to do it in windows 10?

**Comment by andmarti1424** [comment]
Hello. See this https://github.com/andmarti1424/sc-im/issues/460

**Comment by andmarti1424** [comment]
Actually I believe it should be pretty easy to mod and get it built.
You do not need nothing more rather than ncurses, gcc, make, bison..
libzip optional if you need xlsx support.

**Reply by Virgilio-AI** [reply]
I will try it right now, I am little bit new I don't have much experience so I don't promess anything haha
